recipes/mdsrv/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MDsrv" %}
+{% set version = "0.3.5" %}
+{% set sha256 = "3dd218ec0d62cf47061699b8ab33715d6875fad824091f2ab37e546467fef0c3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
Looking at the package's `setup.py` it appears that there is a python `entry_point`. Please add it here:

https://conda.io/docs/building/meta-yaml.html#python-entry-points

https://github.com/arose/mdsrv/blob/master/setup.py#L47-L48